src/App.js

```diff
@@ -5,9 +5,10 @@ import Summary from './views/Summary';
 import Expenses from './views/Expenses';
 import MonthSelector from './components/MonthSelector';
 
+const token = process.env.REACT_APP_MOSTASH_API_KEY;
```

Comment:
Any particular reason why you moved the env variables out of the component? (just personal interest)